src/date.rs

```diff
@@ -204,12 +204,8 @@ impl Display for HttpDate {
             12 => b"Dec",
             _ => unreachable!(),
         };
-        let mut buf: [u8; 29] = [
-            // Too long to write as: b"Thu, 01 Jan 1970 00:00:00 GMT"
```

Comment:
Do you know what this comment was about?

Seems to work fine here.

Reply:
Earlier versions of the rust compiler didn't implement automatic de-reference for long strings (longer than 16 bytes iirc). It still isn't implemented for strings with >32 bytes: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=2a8a1bd41169f4b0f0fa73e3dfca869a

However you can now use `*b"something"` to de-reference  longer strings. Not sure if this wasn't possible back then, not done for performance reasons or I didn't think of it.